composer.json

```diff
@@ -82,7 +82,6 @@
     "require": {
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
-        "marc1706/fast-image-size": "^1.1.0",
```

Comment:
Was this added because of min-testing before?

Reply:
Yes, that's how I understand it from the git history.

Reply:
It looks, we do not need it anym,ore, because of the amp removal

Reply:
`composer why marc1706/fast-image-size
thunder/thunder-develop  3.x-dev  requires  marc1706/fast-image-size (^1.1.0)  
lullabot/amp             1.1.4    requires  marc1706/fast-image-size (1.*)`

Reply:
Ah ok, and because lullabot/amp's requirement was to loose, we had to add our own.

Reply:
exactly, so scrap it

Reply:
So, approve?